skops/cli/_convert.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import pathlib
+import pickle
+from typing import Optional
+
+from skops.cli._utils import get_log_level
+from skops.io import dumps, get_untrusted_types
+
+
+def _convert_file(input_file: os.PathLike, output_file: os.PathLike):
+    """
+    Function that is called by ``skops convert`` entrypoint.
+
+    Loads a pickle model from the input path, converts to skops format, and saves to
+    output file.
+
+    Parameters
+    ----------
+    input_file : os.PathLike
```

Comment:
I honestly think `os.PathLike` is pretty self-documenting, but I could also see a user getting confused by not seeing the type they're expecting in the docstring. 

I'm happy with either way

Reply:
Hard to say if a user who doesn't know the type would guess that `str` is a path-like. Anyway, I won't object to leaving it as is, especially since it means that the type annotation and docstring are identical, I just wanted to bring up the possible friction here.